arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -121,8 +121,8 @@
         "tags": [
           "Registries"
         ],
-        "description": "Creates a container registry with the specified parameters.",
-        "operationId": "Registries_Create",
+        "description": "Creates or updates a container registry with the specified parameters.",
+        "operationId": "Registries_CreateOrUpdate",
```

Comment:
`Webhook` couldn't follow the same model since the request body is a super set of the response body. `Replication` already follows this model.